frontend/components/Footer.vue

```diff
@@ -0,0 +1,81 @@
+<!-- /components/Footer.vue -->
+<template>
+    <footer class="">
+        <div class="mx-auto lg:my-0 w-4/5 p-4 py-6 lg:py-8">
+            <hr class="mt-8 mb-6 border-gray-500 sm:mx-auto lg:mt-6" />
+            <div class="min-[800px]:flex min-[800px]:items-center min-[800px]:justify-between">
+                <div class="flex-wrap md:justify-start md:text-start text-center">
+                    <div class="inline-block mr-10 mb-6 text-sm text-gray-500"><nuxt-link  class="hover:underline" to="/productList">Products</nuxt-link></div>
+                    <div class="inline-block mb-6 text-sm text-gray-500"><nuxt-link  class="hover:underline" to="/eventlist">Events</nuxt-link></div>
+                    <div class="flex md:justify-start justify-center text-xs text-gray-500 items-center"><p class="text-2xl inline mr-2">©</p><nuxt-link  class="hover:underline " to="/">Getit 2024</nuxt-link>. All rights reserved
+                    </div>
+                </div>
+                
+                <nuxt-link class="justify-center text-3xl font-bold flex items-center text-black md:m-4 m-6" to="/">
+                    <img class="h-12"
+                        src="https://raw.githubusercontent.com/bsoc-bitbyte/GetIt/10a0fcc39d52d116428dd49505ead2f597e7a30e/assets/get_it.png"
```

Comment:
hey, we already have the logo in frontend/assets folder, please import it from there

Reply:
Done, actually I took the reference from navbar that why